static/sass/components/_hero.scss

```diff
@@ -25,3 +25,54 @@
     }
   }
 }
+
+// Hero Images
+// ===========
+
+
+.hero {
+  max-height: 35vh;
+  overflow: hidden;
+  position: relative;
+
+  @include above('medium-page') {
+    max-height: 45vh;
+  }
+
+  + .title {
+    background-image: linear-gradient(50deg, color('background') 80%, transparent 80%);
+    margin-bottom: 0;
+    margin-top: negative('double-gutter');
+    padding-bottom: size('shim');
+    padding-right: 20%;
+    padding-top: size('shim');
+    position: relative;
+    z-index: 1;
+
+    @include above('medium-page') {
+      background-image: linear-gradient(50deg, color('background') 75%, transparent 75%);
+      margin-top: negative('spacer');
+      padding-bottom: size('gutter');
+      padding-right: calc(23% - 3vw);
```

Comment:
They are sorta arbitrary ... that is unless we don't use the max-width which I believe we will be not using (again, I kinda thought I would wait for template grid but I suppose I don't have to wait for that).